x-pack/plugin/repository-encrypted/build.gradle

```diff
@@ -0,0 +1,31 @@
+evaluationDependsOn(xpackModule('core'))
+
+apply plugin: 'elasticsearch.esplugin'
+esplugin {
+    name 'repository-encrypted'
+    description 'Elasticsearch Expanded Pack Plugin - client-side encrypted repositories.'
+    classname 'org.elasticsearch.repositories.encrypted.EncryptedRepositoryPlugin'
+    extendedPlugins = ['x-pack-core']
+}
+
+thirdPartyAudit {
+    ignoreViolations (
```

Comment:
we should add a comment (or a TODO to do it) to say why it's ok to ignore those

Reply:
I will add integ tests when everything is put together, we won't merge to master without integ tests for sure. I will add a TODO if you think so.